base_layer/core/src/validation/block_validators.rs

```diff
@@ -41,12 +41,13 @@ use tari_crypto::tari_utilities::hash::Hashable;
 pub const LOG_TARGET: &str = "c::val::block_validators";
 
 /// This validator tests whether a candidate block is internally consistent
-#[derive(Default)]
-pub struct StatelessValidator {}
+pub struct StatelessValidator {
+    consensus_constants: ConsensusConstants,
+}
 
 impl StatelessValidator {
-    pub fn new() -> Self {
-        Self {}
+    pub fn new(consensus_constants: ConsensusConstants) -> Self {
```

Comment:
```suggestion
    pub fn new(consensus_constants: &ConsensusConstants) -> Self {
```

As written, if coming from ConsensusManager, you'd have to write the clunky `::new(cm.consensus_constants().deref().clone())` right?

with a reference, I believe `::new(&cm.consensus_constants()` will work fine.

Reply:
This hides the clone, but will update